src/main/resources/application.yml

```
@@ -2,4 +2,7 @@ spring:
   application:
     name: 'NBA Stats 24-25 '
     username: LaloBall01
-    password: BlackAdam7
\ No newline at end of file
+    password: BlackAdam7
+server:
+  servlet:
+    context-path: /player-service
```

Probablemente el context-path para toda tu api sería mejor:

```suggestion
    context-path: /nba-stats/api/
```

Esto ya que no solo trabajarás con Players, puede que en el futuro también tengas controllers o endpoints de Teams, Stats, etc